src/nextjournal/clerk/render.cljs

```diff
@@ -414,16 +414,12 @@
           (if (string? s) [s] s))))
 
 (defn render-quoted-string [s {:as opts :keys [path viewer !expanded-at] :or {path []}}]
```

Comment:
Took the opportunity to simplify this, assuming presented nodes with string values will always give just a string value. Is this correct?